### PR TITLE
RunbookScanAgent and Command Alias Changes

### PR DESCRIPTION
**Pull request by v-nitja**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Listeners/UserReports/WritePsConsole.ps1 [hunk 1]
```diff
@@ -472,8 +474,60 @@ class WritePsConsole: FileOutputBase
...
+
+                $myObject | Add-Member -type NoteProperty -name Name -Value $_
+                $myObject | Add-Member -type NoteProperty -name Alias -Value $key
+                $myObject | Add-Member -type NoteProperty -name Value -Value $val
+
+                $paramlist += $myObject
+            }
+
+            $aliasshort = $aliasName.ToLower()
+            $cmID = "$aliasshort "
```
Comment (sbyna-msft) [inline src/AzSK/Framework/Listeners/UserReports/WritePsConsole.ps1]: @v-nitja  add some comments to this logic

Comment (sbyna-msft) [reply src/AzSK/Framework/Listeners/UserReports/WritePsConsole.ps1]: And can you move all this logic to some function and just call that function here?

File: src/AzSK/Framework/Listeners/UserReports/WritePsConsole.ps1 [hunk 2]
```diff
@@ -472,13 +474,83 @@ class WritePsConsole: FileOutputBase
 				$this.WriteMessageData($_);
 			}
 		}
-
-		$this.WriteMessage("Method Name: $methodName `r`nInput Parameters: $(($this.InvocationContext.BoundParameters | Out-String).TrimEnd()) `r`n" + [Constants]::DoubleDashLine , [MessageType]::Info);                           
+        
+        if($aliasName)
+        {
+            $aliasName = $aliasName.ShortName 
+            $aliasName = $aliasName
```
Comment (ShrutiBhambhani) [inline src/AzSK/Framework/Listeners/UserReports/WritePsConsole.ps1]: line 481 can be removed

File: src/AzSK/Framework/Listeners/UserReports/WritePsConsole.ps1 [hunk 3]
```diff
@@ -472,13 +474,83 @@ class WritePsConsole: FileOutputBase
...
+            foreach($item in $paramlist)
+            {
+                $ky = $item.Alias
+                $vl = $item.Value
+                if($ky)
+                {
+                    if($vl -eq $true)
+                    {
+                        $vl = ""
+                    }
```
Comment (ShrutiBhambhani) [inline src/AzSK/Framework/Listeners/UserReports/WritePsConsole.ps1]: if($vl -eq $true) block can be moved above if($ky) block

**Pull request merged**